### DISCUSSION TITLE
[BP#93] : Stocker les données dans le cloud

### DISCUSSION THREAD

**Discussion by Altheachevalier** [cnumr/best-practices#91] [Bonnes pratiques] [answered]
Pour cet best practice, je suis en accord néanmoins il est important de choisir un fournisseur/hébergeur responsable (comme ovh) et ainsi cela va avec l'autre best practice sur le choix d'un hebergeur vert. 
Peux être faire une seule best practice pour les deux ? 

Egalement, le choix du one premise s'il est bien géré et écoresponsable peut être totalement écologique. 
J'avais parlé à une entreprise qui avait des datacenter et qui chauffait une piscine et une autre entreprise qui cultivait des ténébrions dans les datacenter afin de faire une "agriculture responsable" en même temps. 
Effectivement la température des datacenters est excellente pour la culture des ténébrions.

**Comment by ACTLEM** [comment, marked as answer]
Personnellement, je pense qu'il faut garder les deux bonnes pratiques car le plus de gros de l'impact concernant le stockage est sur la fabrication des disques durs (HDD ou SSD). Certes avoir un hébergeur responsable est mieux mais mutualiser le stockage (quelque soit le cloud provider) est déjà un bon premier pas.
